configuration/pih/scripts/global/zl.js

```diff
@@ -1,74 +1,101 @@
 function setUpExpandableContacts(badPhoneNumberMsg) {
-  var contactsToShow = 1;
-  var elem = document.getElementsByClassName("contacts");
-  var maxContacts = elem.length;
+    var contactsToShow = 1;
+    var elem = document.getElementsByClassName("contacts");
+    var maxContacts = elem.length;
```

Comment:
@brandones @louidorjp  looks like all this is reformating the tabs to be 4 spaces vs 2 spaces.

Do we have a sense of what the standard is for JavaScript? I don't have a strong preference, but I figure we should all do the same so we all aren't reformatting every time we commit.  Brandon, what does OpenMRS 3.0 use?

Also, Brandon suggested we add an .editorconfig file to start enforcing this automatically... do one of you want to look into setting that up?

Reply:
OpenMRS 3.0 uses 2-space indents. This is pretty standard. See e.g. [Google JS Style Guide](https://google.github.io/styleguide/jsguide.html).

Let's just get this PR in and I'll add the editorconfig later.

Reply:
@mogoodrich  I have'nt merge privileges to the ZL repo